docs/sdks/swift-sdk/integrate-with-okta.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Integrate With Okta
+sidebar_position: 2
+---
+
+This guide describes how to configure Okta to delegate to Beyond Identity for authentication during an OAuth2 authorization flow.
+
+### Prerequisites
+
+ - [Integrate With Okta](../../sso-integrations/integrate-with-okta)
+ - [Swift SDK Overview](overview)
+
+Before calling [`Embedded.shared.authenticate`](overview#authentication), we must [Authenticate With Okta](integrate-with-okta#authorize-with-okta)
+
+### Authorize With Okta
+
+ - Step 1: Okta Authorize URL
```

Comment:
Also noticed this on Will's documentation:

I realized there's actually a step missing here before step 1, configuring the authenticator config to:

- be set to type "Embedded"
- set the invoke_url to the app_scheme or universal url/App link of your application

You can also point to https://developer-docs-git-v1-beyondidentity.vercel.app/docs/v1/platform-overview/authenticator-config#embedded for more context on "Embedded" authenticator config.

Reply:
sure, added